projects/sunbird-pdf-player/src/lib/sunbird-pdf-player.component.ts

```diff
@@ -39,6 +40,7 @@ export class SunbirdPdfPlayerComponent implements OnInit, OnDestroy, OnChanges,
   @Input() action: string;
   @Output() playerEvent: EventEmitter<object>;
   @Output() telemetryEvent: EventEmitter<any> = new EventEmitter<any>();
+  @Output() playNextContent: EventEmitter<any> = new EventEmitter<any>();
```

Comment:
Keep this as part of player event  and also raise telemetry event for this click as interact event